3-enrich/scala-common-enrich/src/test/scala/com.snowplowanalytics.snowplow.enrich.common/enrichments/registry/EnrichmentConfigsSpec.scala

```diff
@@ -457,8 +457,7 @@ class EnrichmentConfigsSpec extends Specification with ValidationMatchers with D
             "uri": "https://example.com"
           },
           "includeUseragentFile": {
-             "database": "include_current.txt",
-             "uri": "invalid\\uri"
+             "database": "include_current.txt"
```

Comment:
can't we create an invalid uri?

Reply:
if not we need to remove this test

Reply:
It was not so easy to find an invalid one with the relaxed parsing